setup.py

```diff
@@ -13,6 +13,7 @@
     license="MIT",
     entry_points={
         "console_scripts": [
+            "micro_sam.annotator = micro_sam.sam_annotator.annotator:main",
```

Comment:
@constantinpape, will this entry not be part in the conda-forge review?

Reply:
It will be part of conda-forge, but I will need to draft a new release of `micro_sam` first. Will try to do that tomorrow morning.

Reply:
Thanks. Got it.